### PR TITLE
add Aerofox AYK320 drone param config file

### PR DESCRIPTION
**Pull request by AEROFOXtech**
add Aerofox new drone param config file

### CODE REVIEW

File: Tools/Frame_params/Aerofox_AYK320.param [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Q_A_RAT_YAW_FLTE 2
+Q_A_RAT_YAW_FLTT 12.5
+Q_A_RAT_YAW_P    1
+Q_A_SLEW_YAW     3000
+Q_LAND_FINAL_ALT 12
+Q_LAND_SPEED     40
+Q_M_BAT_IDX      1
+Q_M_BAT_VOLT_MAX 50.4
+Q_M_BAT_VOLT_MIN 40
+Q_M_THST_HOVER   0.338
```
Comment (rishabsingh3003) [inline Tools/Frame_params/Aerofox_AYK320.param:87]: Q_M_THST_EXPO isn't set?

**Comment by rmackay9** [comment]
txs for the contribution.

By the way, it would be good to add a "Tools:" prefix to the commit... just to be clear, I mean the commit title, not the PR title.

File: Tools/Frame_params/Aerofox_AYK320.param (hunk 1 above)
Comment (AEROFOXtech) [reply Tools/Frame_params/Aerofox_AYK320.param:87]: Yes, in our tests, this model of drone has a hover throttle of exactly 3.5 when there is no load.

**Comment by rishabsingh3003** [comment]
@AEROFOXtech the commits do not look right to me. They need to be squashed into a single commit with the commit message being: "Tools: add Aerofox AYK320 drone param config file". Right now you have two commits and the wrong commit message

**Pull request closed without merging**